docs/source/api/_devices.rst

```diff
@@ -150,6 +150,7 @@ Slits
 
 .. autosummary::
```

Comment:
Documentation will look like this:
![image](https://github.com/BCDA-APS/apstools/assets/2279984/a7b754e6-3dc4-4917-aab2-6acedaa930a9)

The way to get documentation for the `FCpt` components is to add docstrings to each?  Not sure it's worth it now.  Inclined to skip it.

Reply:
Nothing to be changed here. Looks good!